sfe/sfe.go

```diff
@@ -160,20 +174,30 @@ func (sfe *SelfServiceFrontEndImpl) UnpauseForm(response http.ResponseWriter, re
 	sfe.renderTemplate(response, "unpause-form.html", tmplData{unpausePostForm, incomingJWT, regID, identifiers})
 }
 
-// UnpauseSubmit serves a page indicating if the unpause form submission
-// succeeded or failed upon clicking the unpause button. We are explicitly
-// choosing to not address CSRF at this time because we control creation and
-// redemption of the JWT.
+// UnpauseSubmit serves a page showing the result of the unpause form submission.
+// CSRF is not addressed as we control JWT creation and redemption.
```

Comment:
I know this comment was already here, but it's subtly wrong. Mind tweaking it while you're here?

```suggestion
// CSRF is not addressed because a third party causing submission of an unpause form is not harmful.
```